dotCMS/src/main/java/com/dotcms/graphql/exception/TypeGenerationException.java

```diff
@@ -0,0 +1,23 @@
+package com.dotcms.graphql.exception;
+
+import com.dotmarketing.exception.DotRuntimeException;
+import graphql.schema.GraphQLType;
+
+
+/**
+ * Runtime Exception thrown when a {@link GraphQLType} can't be generated
+ */
+public class TypeGenerationException extends DotRuntimeException {
+
+    public TypeGenerationException(String message) {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Parameter 'message' is not assigned and could be declared final](https://app.codacy.com/manual/dotCMS/core/pullRequest?prid=5642169)